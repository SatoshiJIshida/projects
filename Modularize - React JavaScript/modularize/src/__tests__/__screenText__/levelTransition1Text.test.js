import React from "react";
import { shallow } from "enzyme";
import LevelTransition1Text from "../../components/screenText/levelTransition1Text";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Level Transition Text (1)", () => {
  it("should be defined", () => {
    expect(LevelTransition1Text).toBeDefined();
  });

  it("should render correctly", () => {
    const levelTransition1Text = shallow(
      <LevelTransition1Text name="levelTransition1Text-test" />
    );
    expect(levelTransition1Text).toMatchSnapshot();
  });
});
