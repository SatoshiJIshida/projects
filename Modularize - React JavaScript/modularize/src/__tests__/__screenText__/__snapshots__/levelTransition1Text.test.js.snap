// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level Transition Text (1) should render correctly 1`] = `ShallowWrapper {}`;
